frontend/src/components/Admin/TeamEvent/TeamEventDashboard.tsx

```diff
@@ -51,11 +52,58 @@ const TeamEventDashboard: React.FC = () => {
 
   if (isLoading) return <Loader active>Fetching team event data...</Loader>;
 
+  const handleExportToCsv = () => {
+    const csvData = allMembers.map((member) => {
+      const totalCredits = teamEvents.reduce(
+        (val, event) => val + calculateTotalCreditsForEvent(member, event),
+        0
+      );
+      const initiativeCredits = teamEvents.reduce(
+        (val, event) => val + calculateInitiativeCreditsForEvent(member, event),
+        0
+      );
```

Comment:
The previous two functions are the same as the ones found on lines 119-122 and 150-157. I think it would be good to refactor them to something like `getAllCredits(member)` and `getAllInitiativeCredits(member)`, or something like that for code reusability.